content/en/docs/installation/supported-releases.md

```diff
@@ -182,13 +183,13 @@ maintainers think is reasonable to support and to test.
 
 Our testing coverage is:
 
-| Release branch | Prow configuration            | Dashboard                 | Kubernetes versions tested   |  Periodicity  |
-| :------------: | :---------------------------- | :------------------------ | :--------------------------- | :-----------: |
-|      PRs       | [`presubmits.yaml`][]         | [`presubmits-blocking`][] | 1.21                         |  On each PR   |
-|     master     | [`periodics.yaml`][]          | [`master`][]              | 1.16, 1.17, 1.18, 1.19, 1.21 | Every 2 hours |
-|  release-1.5   | [`next-periodics.yaml`][]     | [`next`][]                | 1.16, 1.17, 1.18, 1.19, 1.21 | Every 2 hours |
-|  release-1.4   | [`previous-periodics.yaml`][] | [`previous`][]            | 1.16, 1.17, 1.18, 1.19, 1.21 | Every 2 hours |
-|  release-1.3   | N/A                           |                           | N/A                          |      N/A      |
+| Release branch | Prow configuration            | Dashboard                 | Kubernetes versions tested  |  Periodicity  |
+| :------------: | :---------------------------- | :------------------------ | :-------------------------: | :-----------: |
+|      PRs       | [`presubmits.yaml`][]         | [`presubmits-blocking`][] |         1.21, 1.22          |  On each PR   |
+|     master     | [`periodics.yaml`][]          | [`master`][]              |         1.16 → 1.22         | Every 2 hours |
+|  release-1.6   | [`next-periodics.yaml`][]     | [`next`][]                |         1.16 → 1.22         | Every 2 hours |
+|  release-1.5   | [`previous-periodics.yaml`][] | [`previous`][]            |         1.16 → 1.22         | Every 2 hours |
+|  release-1.4   | N/A                           |                           |             N/A             |      N/A      |
```

Comment:
note 1: the original was missing k8s 1.20 on the versions tested, but we _do_ test with 1.20 on the periodics.

note 2: I guessed which version of k8s we'll be testing against for PRs would be 1.21 and 1.22, but this may need changing